build/banner.js

```diff
@@ -1,7 +0,0 @@
-export function createBanner(version) {
```

Comment:
I've moved the `createBanner` function into the Rollup config as that's the only place it's being used.